tools/mtouch/Application.cs

```diff
@@ -1995,6 +1995,10 @@ public void BuildMSymDirectory ()
 			foreach (var target in Targets) {
 				GenerateMSymManifest (target, target_directory);
 				var msymdir = Path.Combine (target.BuildDirectory, "Msym");
+				if (!Directory.Exists (msymdir)) {
+					ErrorHelper.Warning (97, $"Aot files could not be found at the expected directory '{msymdir}'.");
```

Comment:
Better but still confusing, the _important_ (executable) AOT files are present - else the app would not work.
`Aot files` -> `AOT data files` ?
or maybe just
`The directory {msymdir} containing the mono symbols could not be found.` ?